crates/jstz_node/src/main.rs

```diff
@@ -32,44 +27,18 @@ struct Args {
     #[arg(short, long)]
     rollup_endpoint: Option<String>,
 
-    #[arg(long)]
+    #[arg(long, default_value = DEFAULT_KERNEL_FILE_PATH)]
     kernel_file_path: String,
 }
 
 #[actix_web::main]
```

Comment:
To avoid issues in future, we should explicitly switch to `#[tokio::main]` (which `actix` uses internally)